### PR TITLE
Update krew install doc

### PR DESCRIPTION
**Pull request by astraw99**
To make the installation simpler.

fixes https://github.com/kvaps/kubectl-node-shell/issues/61

### CODE REVIEW

**Comment by astraw99** [comment]
@kvaps PTAL thanks.

**Comment by astraw99** [comment]
ping @kvaps

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ Start a root shell in the node's host OS running.
 
 using [krew](https://krew.sigs.k8s.io/):
 
+<pre>
```
Comment (kvaps) [inline README.md:12]:
```suggestion
Plugin can be installed from the official krew repository:

<pre>
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ Start a root shell in the node's host OS running.
...
+kubectl krew install node-shell
+</pre>
+
+If the plugin not found, install as follows:
```
Comment (kvaps) [inline README.md:16]:
```suggestion
Or from our own krew repository:
```

**Review by kvaps — COMMENTED** [review]
Sorry for long delay

I just think it would be better to clarify what exactly is going on here

**Review by kvaps — APPROVED** [review]

**Comment by kvaps** [comment]
Thank you, merged

**Pull request merged**